From: Alessandro Ghedini <al3xbio@gmail.com>
Date: Fri, 2 Feb 2024 14:39:04 +0500
Subject: Add faust include dir to CFLAGS to fix FTBS.

Bug-Debian: http://bugs.debian.org/642662
Origin: vendor
Forwarded: http://code.google.com/p/foo-yc20/issues/detail?id=4
Last-Update: 2011-11-01
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 44a7832..f98e2e2 100644
--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@ CFLAGS=-O2 -ffast-math -ftree-vectorize
 endif
 endif
 
-CFLAGS_X = $(CFLAGS) -fPIC -DVERSION=$(VERSION) -Isrc/ -Iinclude/ -DPREFIX=$(PREFIX) -Wall
+CFLAGS_X = $(CFLAGS) -fPIC -DVERSION=$(VERSION) -I/usr/lib/faust -Isrc/ -Iinclude/ -DPREFIX=$(PREFIX) -Wall
 
 ifneq ($(findstring MINGW, $(OS)),)
 CFLAGS_X += -D_USE_MATH_DEFINES=1
