Description: Fix "Pitch" MIDI control number as per README
Origin: vendor
Bug-Debian: http://bugs.debian.org/712114
Forwarded: no
Author: Alex Volkov <skolwork@bk.ru>
Reviewed-by: Alessandro Ghedini <ghedo@debian.org>
Last-Update: 2014-02-28

diff -Nurbp a/src/foo-yc20.cpp b/src/foo-yc20.cpp
--- a/src/foo-yc20.cpp	2011-01-28 11:22:38.000000000 +0500
+++ b/src/foo-yc20.cpp	2013-06-13 07:45:20.000000000 +0600
@@ -93,7 +93,7 @@ YC20Processor::YC20Processor()
 	: processor(0)
 	, ui(0)
 {
-	controlPerLabel["pitch"]       = new Control(5, -1.0, 1.0);
+	controlPerLabel["pitch"]       = new Control(50, -1.0, 1.0);
 	controlPerLabel["volume"]      = new Control(7);
 	controlPerLabel["bass volume"] = new Control(51);
 	
